src/pages/Dashboard/Dashboard.component.tsx

```diff
@@ -25,8 +30,21 @@ export const DashboardComponent = ({
   totalUSD,
   fishBalance,
   fishVesting,
-}: DashboardComponentProps) => (
+}: DashboardComponentProps) => {
+  
+  const [showDialog, setShowDialog] = useState(false);
+
+  return (
   <PageAligner>
+    <AppDialog 
```

Comment:
I'm not sure if Dashboard is a good place to put this. You can leave it in the `App.tsx` component for now. We're putting all our components there temporary